components/MainColor/index.js

```diff
@@ -1,8 +1,27 @@
-export default function MainColor({ text, id }) {
+import { Fragment } from "react";
+import { StyledInput, StyledDiv, StyledLabel } from "./styles";
+
+export default function MainColor({ color, handleColorChange }) {
+  const colors = [
+    { id: "color1", text: "Color1", name: "color1" },
+    { id: "color2", text: "Color2", name: "color2" },
+    { id: "color3", text: "Color3", name: "color3" },
+  ];
+
```

Comment:
why do you have two color arrays (here and in your app.js? you could combine them and start with a color array with example/start colors and then change only one array instead of working with two, that are also called the same name: "colors"

Reply:
For now I want to have the Array above my mapping. But I changed the name of the Array, so the difference is more clear now